kvmagent/kvmagent/test/utils/vm_utils.py

```
@@ -125,7 +126,20 @@ def get_bootMode():
     },
     "x2apic": True,
     "acpi": True,
-    "vmCpuQuota": 1000000
+    "vmCpuQuota": 1000000,
+    "cdRoms": {
```

Ruff (0.12.2)

Dictionary key literal `"cdRoms"` repeated

(F601)